### PR TITLE
Add basic proof of concept with genkit and a basic prompt analysis

### PR DESCRIPTION
**Pull request by arbrown**

### CODE REVIEW

File: game-advisor/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+lib/
```
Comment (moficodes) [inline game-advisor/.gitignore:1]:
```suggestion
lib/

```

File: game-advisor/prompts/summary.prompt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+improve their performance, and a letter grade for the overall performance.
+
+An A+ game will have at least 10 loops and at least 40 flippers hit.
+
+Do not output markdown or any explanation or preamble.  Simply output the JSON in the schema to be specified below.
+
+Do not output code to generate JSON.  Output JSON ONLY
+
+Game Log:
+{{gameLog}}
```
Comment (moficodes) [inline game-advisor/prompts/summary.prompt:30]:
```suggestion
{{gameLog}}

```

File: game-advisor/prompts/summary.prompt [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+You are an expert on pinball gameplay.  
+Read the following gameplay log and provide an analysis of the game.  
+You should provide a short 5-15 word headline summarizing the game,
+a more-indepth analysis of the game, 3-5 tips for how the player could 
+improve their performance, and a letter grade for the overall performance.
+
+An A+ game will have at least 10 loops and at least 40 flippers hit.
+
+Do not output markdown or any explanation or preamble.  Simply output the JSON in the schema to be specified below.
```
Comment (moficodes) [inline game-advisor/prompts/summary.prompt:25]: Do we know if the schema gets injected before or after?

Should we just say
```
Simply output the JSON in the Schema specified
```

**Review by moficodes — COMMENTED** [review]
Couple of cleanup and one question.

Other wise looks good!

File: game-advisor/prompts/summary.prompt (hunk 3 above)
Comment (arbrown) [reply game-advisor/prompts/summary.prompt:25]: The schema is output after the prompt (by genkit).  It's all just a big bundle of text, though maybe genkit does something else with other model plugins.

In any case, this line helped it more consistently follow the schema, but I'm sure we'll continue to tweak it.

**Review by moficodes — APPROVED** [review]

**Pull request merged**